src/Text/Pandoc/Options.hs

```diff
@@ -363,6 +363,7 @@ data WriterOptions = WriterOptions
   , writerBeamer           :: Bool       -- ^ Produce beamer LaTeX slide show
   , writerSlideLevel       :: Maybe Int  -- ^ Force header level of slides
   , writerChapters         :: Bool       -- ^ Use "chapter" for top-level sects
+  , writerParts            :: Bool       -- ^ Use "part" for top-level sects in LaTeX
```

Comment:
Of course, if we touch anything in WriterOptions, we'll
need to increment the second version number (1.18).
So we might consider doing another 1.17.* refresh
before merging this change.